docs/en/ingest-management/integrations/upgrade-integration.asciidoc

```diff
@@ -16,9 +16,9 @@ TIP: In larger deployments, you should test integration upgrades on a sample
 [[upgrade-integration-to-latest-version]]
 == Upgrade an integration to the latest version
 
-. In {kib}, go to *Management > Integrations > Installed*. Search for and select
-the integration you want to upgrade. Notice there is a warning icon next to the
-version number to indicate a new version is available.
+. In {kib}, go to *Management > Integrations > Installed integrations*. Search
+for and select the integration you want to upgrade. Notice there is a warning
+icon next to the version number to indicate a new version is available.
```

Comment:
@dedemorton it'd be nice to add a reminder to upgrade standalone agents in this section